server/svix-server/src/v1/endpoints/application.rs

```diff
@@ -84,24 +89,30 @@ pub struct ApplicationPatch {
     #[serde(default, skip_serializing_if = "UnrequiredNullableField::is_absent")]
     #[validate]
     pub uid: UnrequiredNullableField<ApplicationUid>,
+
+    #[serde(default, skip_serializing_if = "UnrequiredField::is_absent")]
+    pub metadata: UnrequiredField<Metadata>,
 }
 
 impl ModelIn for ApplicationPatch {
-    type ActiveModel = application::ActiveModel;
+    type ActiveModel = (application::ActiveModel, applicationmetadata::ActiveModel);
 
-    fn update_model(self, model: &mut Self::ActiveModel) {
+    fn update_model(self, (app, app_metadata): &mut Self::ActiveModel) {
```

Comment:
This syntax looks a bit weird, does it even work? What's the type of app? Metadata?

Reply:
Oh, I just saw it, it's very confusing.

Reply:
I feel like it just needs to be split, there's no real value merging them both into this `update_model`, as metadata is really just a one field, no?